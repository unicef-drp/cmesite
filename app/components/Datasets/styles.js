import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  wrapper: {
    backgroundColor: theme.palette.primary.main,
    paddingTop: theme.spacing(4),
    paddingBottom: theme.spacing(4),
  },
  list: {
    paddingTop: theme.spacing(2),
  },
  item: {
    borderRadius: 0,
  },
  typo: {
    paddingLeft: theme.spacing(2),
    textTransform: 'none',
  },
  icon: {
    color: theme.palette.secondary.main,
  },
}));
