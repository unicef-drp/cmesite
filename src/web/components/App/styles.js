export default {
  root: {
    width: '100%',
    height: '100%',
    overflowX: 'hidden',
    overflowY: 'scroll',
    backgroundColor: ({ theme }) => theme.palette.background.contentFrame,
    paddingTop: ({ theme }) => theme.mixins.toolbar.minHeight,
  },
  container: {},
};
