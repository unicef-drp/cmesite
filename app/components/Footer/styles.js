import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  container: {
    paddingTop: theme.spacing(2),
    paddingBottom: theme.spacing(2),
    display: 'flex',
    justifyContent: 'space-between',
    alignItems: 'center',
    flexWrap: 'wrap',
    [theme.breakpoints.down('xs')]: {
      justifyContent: 'center',
    },

    // sticky footer -> https://philipwalton.com/articles/normalizing-cross-browser-flexbox-bugs/
    flexShrink: 0,
  },
  item: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    flexWrap: 'wrap',
  },
}));
