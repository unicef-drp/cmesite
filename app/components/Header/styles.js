import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  toolbar: {
    [theme.breakpoints.down('xs')]: {
      display: 'flex',
      justifyContent: 'space-between',
    },
  },
  logo: {
    height: 30,
    paddingRight: theme.spacing(2),
  },
  typo: {
    color: theme.palette.primary.dark,
  },
  titlebar: {
    backgroundColor: theme.palette.secondary.main,
  },
  menubar: {
    top: theme.spacing(8),
  },
  selectedMenu: {
    border: `1px solid ${theme.palette.secondary.main}`,
    color: `${theme.palette.secondary.main}!important`,
  },
  menu: {
    paddingLeft: theme.spacing(2),
    paddingRight: theme.spacing(2),
  },
}));
