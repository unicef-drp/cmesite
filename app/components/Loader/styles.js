import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  root: {
    paddingTop: theme.spacing(4),
    paddingBottom: theme.spacing(4),
  }
}));
