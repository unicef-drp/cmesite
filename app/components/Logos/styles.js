import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  igmeLogo: {
    height: 30,
    marginLeft: 0,
    marginRight: theme.spacing(),
  },
  logo: {
    height: 50,
    marginTop: theme.spacing(),
    marginBottom: theme.spacing(),
    marginLeft: theme.spacing(),
    marginRight: theme.spacing(),
  },
  big: {
    height: 100,
  },
}));
