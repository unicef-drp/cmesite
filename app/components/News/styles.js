import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  wrapper: {
    backgroundColor: theme.palette.secondary.main,
    marginTop: theme.spacing(-6),
    paddingTop: theme.spacing(2),
    paddingLeft: theme.spacing(4),
    paddingRight: theme.spacing(4),
    height: `calc(100% + ${theme.spacing(6)}px)`,
  },
  container: {
    paddingTop: theme.spacing(1),
    [theme.breakpoints.down('sm')]: {
      display: 'flex',
      justifyContent: 'space-around',
    },
  },
  card: {
    borderRadius: 0,
    background: 'none',
    padding: 0,
    [theme.breakpoints.down('sm')]: {
      width: 320,
      paddingTop: 0,
      paddingLeft: theme.spacing(1),
      paddingRight: theme.spacing(1),
    },
  },
  media: {
    [theme.breakpoints.up('md')]: {
      paddingTop: '56.5%',
      height: 0,
    },
    [theme.breakpoints.down('sm')]: {
      height: 160,
    },
  },
  content: {
    paddingLeft: 0,
    paddingRight: 0,
  },
  typo: {
    color: theme.palette.primary.dark,
  },
}));
