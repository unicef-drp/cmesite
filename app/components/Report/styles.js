import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  card: {
    display: 'flex',
    flexDirection: 'row',
    backgroundColor: theme.palette.primary.light,
    margin: theme.spacing(2),
    width: 340,
    minHeight: 180,
  },
  secondaryCard: {
    backgroundColor: theme.palette.secondary.dark,
    margin: 0,
    marginBottom: theme.spacing(2),
  },
  content: {
    backgroundColor: theme.palette.secondary.main,
    border: `1px solid ${theme.palette.primary.light}`,
  },
  secondaryContent: {
    backgroundColor: theme.palette.secondary.dark,
    border: 'none',
  },
  media: {
    margin: theme.spacing(2),
    paddingLeft: '35%',
    width: 0,
    height: 180,
  },
  typo: {
    color: theme.palette.primary.dark,
  },
  leftIcon: {
    marginRight: theme.spacing(1),
  },
}));
