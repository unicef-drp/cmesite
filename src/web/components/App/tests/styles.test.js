import styles from '../styles';

describe('app | containers | App | styles', () => {
  it('should be defined', () => {
    expect(styles).toBeDefined();
  });

  it('should match snapshot', () => {
    expect(styles).toMatchSnapshot();
  });
});
