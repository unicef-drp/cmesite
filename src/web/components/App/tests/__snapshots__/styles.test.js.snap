// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app | containers | App | styles should match snapshot 1`] = `
Object {
  "container": Object {},
  "root": Object {
    "backgroundColor": [Function],
    "height": "100%",
    "overflowX": "hidden",
    "overflowY": "scroll",
    "paddingTop": [Function],
    "width": "100%",
  },
}
`;
